### PR TITLE
Fixed poll with less than 2 options

### PR DESCRIPTION
**Pull request by Shadran**
# This pull request changes the following things:

```
Fixed poll with less than 2 options
Added log for message reaction error  
Decreased spoiler delay  
Fixed spoiler sizing bug  
Fixed various formatting errors with empty poll title
```

## By submitting this Pull Request I agree to have done the following:

- [x] Run `go fmt` on all the files I've changed
- [x] Tested the bot on my own server to verify the feature works
- [x] Verified with CamD67 that the feature is desired (Check out the Projects page, or submit an issue for new features)
- [x] Assigned CamD67 as the reviewer + Assignee for the PR

@camd67

### CODE REVIEW

File: moebot_bot/util/gifmaker.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ const (
 	xBorder   = 30
 	yBorder   = 14
 	maxWidth  = 500
+	viewDelay = 150 // 1/100th of second
```
Comment (camd67) [inline moebot_bot/util/gifmaker.go:20]: 1/100th of a second?
I think you mean that it's in milliseconds, which would be a bit clearer. Not a big deal.

(Ignore this I realized what you meant by this. That's super weird that it's in 10 millisecond intervals....)

File: moebot_bot/util/util.go [hunk 2]
```diff
@@ -132,9 +137,22 @@ func OpenPollMessage(poll *db.Poll, user *discordgo.User) string {
 func ClosePollMessage(poll *db.Poll, user *discordgo.User) string {
 	var message string
 	if poll.Open {
-		message = user.Mention() + " closed " + UserIdToMention(poll.UserUid) + "'s poll **" + poll.Title + "**!\n"
+		if user.ID == poll.UserUid {
+			message = user.Mention() + " closed his poll"
```
Comment (camd67) [inline moebot_bot/util/util.go]: Let's make sure to use they/their/them when referring to users, especially to be consistent with the rest of the user messaging

Comment (Shadran) [reply moebot_bot/util/util.go]: Fixed with https://github.com/camd67/moebot/pull/12/commits/754546d6bedae618648b940ae5ece6cae1042573

**Comment by camd67** [comment]
👍

**Pull request merged**